src/microbe_stage/PatchMap.cs

```diff
@@ -4,6 +4,39 @@
 using Godot;
 using Newtonsoft.Json;
 
+/// <summary>
+///   Whether the fog of war should be disabled,
+///   mark patches adjacent to the player as explored,
+///   or mark only the patch the player has entered as explored
+/// </summary>
+public enum FogOfWarMode
```

Comment:
The first type in a file should match the filename. This means that this enum needs to be put in a separate file and MapElementVisibility should be put inside the class (if it isn't used widely outside this class, and if it is then it also needs to go int oa separate file).

Reply:
Yeah, MapElementVisibility is an enum I use in PatchRegion, Patch and PatchNode (or whatever that class is called). It is basically a way for me to standardise how each of those elements stores their visibility state